pkg/sbom/spdx/marshal.go

```diff
@@ -105,6 +107,20 @@ func NewMarshaler(opts ...marshalOption) *Marshaler {
 	return m
 }
 
+// The function augmentSpdxData updates each package in packages key,
+// ensuring the spdx json is valid as per https://tools.spdx.org/app/validate/
+// The following keys are being updated
+//  1. licenseConcluded (incorrect delimiter and string value throws error)
+//  2. licenseDeclared (incorrect delimiter and string value throws error)
+//  3. copyrightText (throws a warning if the value is empty)
+//  4. downloadLocation (throws a warning if the value is empty)
+func augmentSpdxData(p *spdx.Package2_2) {
```

Comment:
Would it make sense to update ```pkgToSpdxPackage``` where these properties are being set, instead of reiterating? We can also skip getting license info.

Reply:
There are 3 methods that return spdx.Package2_2 object
1. rootPackage
2. resultToSpdxPackage
3. pkgToSpdxPackage
The idea was to keep the changes in a single place. A revisit these changes with a proper fix in a later sprint

Reply:
ok